Demo/build.gradle

```diff
@@ -94,34 +93,29 @@ dependencies {
     implementation project(':Venmo')
     implementation project(':VisaCheckout')
 
-    implementation 'com.google.code.gson:gson:2.10.1'
-    implementation 'de.greenrobot:eventbus:2.4.1'
-    implementation deps.appCompat
-    implementation 'androidx.preference:preference:1.2.1'
-    implementation 'com.google.android.material:material:1.10.0'
-    implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'androidx.multidex:multidex:2.0.1'
-
-    implementation 'io.card:android-sdk:5.5.1'
-    implementation 'com.braintreepayments:card-form:5.4.0'
```

Comment:
These weren't referenced anywhere. Are they safe to remove?

Reply:
Should be fine to remove. `io.card` is the old snap a photo autofill credit card thing used in the Drop-in and `card-form` is used by the Drop-in as well. Neither should be needed here.

Reply:
Hmm, I'm now seeing errors when removing `card-form` 🤔 there are references in the demo app. I'll leave it in for now, but it sounds like we could refactor it out at some point.